plugin/mutate/source/dextool/plugin/mutate/backend/mutation_type/package.d

```diff
@@ -64,9 +64,15 @@ MutationKind toUser(Mutation.Kind k) @safe nothrow {
     return fromInteralKindToUserKind[k];
 }
 
+//Convert kind to its description
+string toDescription(MutationKind kind) @safe nothrow{
+    return mutationDescription[kind];
+}
+
 private:
 
 immutable MutationKind[Mutation.Kind] fromInteralKindToUserKind;
+immutable string[MutationKind] mutationDescription;
```

Comment:
might as well make it public. then the toDescription function isn't needed.
then when iterating over the keys you can use ".byKeyValue" instead of EnumMembers